src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int MAX_RANDOM_NUMBER = 10;
+    public static final int DEFAULT_RADIUS = 10;
```

Comment:
do you use these constants outside this class?

Reply:
replaced by private
done